src/main/java/com/dataloom/edm/internal/PropertyType.java

```diff
@@ -27,51 +28,71 @@ public PropertyType(
             @JsonProperty( SerializationConstants.TITLE_FIELD ) String title,
             @JsonProperty( SerializationConstants.DESCRIPTION_FIELD ) Optional<String> description,
             @JsonProperty( SerializationConstants.SCHEMAS ) Set<FullQualifiedName> schemas,
-            @JsonProperty( SerializationConstants.DATATYPE_FIELD ) EdmPrimitiveTypeKind datatype ) {
+            @JsonProperty( SerializationConstants.DATATYPE_FIELD ) EdmPrimitiveTypeKind datatype,
+            @JsonProperty( SerializationConstants.PII_FIELD ) boolean piiField ) {
```

Comment:
It would. Unfortunately there can only be one constructor marked with JsonCreator, so if we need to give users the flexibility to specify whether it's PII, we have to make this change.